TACC/protx
# Task/COOKS-75: add texas boundary to map

## Overview: ##

Add boundary of Texas to map

## Related Jira tickets: ##

* [COOKS-75](https://jira.tacc.utexas.edu/browse/COOKS-75)

## Summary of Changes: ##
* Add outline of Texas
* Refactor the requesting of various map data to make requests concurrently 

## Testing Steps: ##
1. Switch to show zip codes and select a demographic feature that is outside of Texas.  
2. Ensure boundary is on top.  
3. Ensure you can't navigate outside of Texas.

## UI Photos:
![Screen Shot 2021-07-12 at 3 18 00 PM](https://user-images.githubusercontent.com/8287580/125351143-54be8800-e325-11eb-8d2f-8ae56ed4a2af.png)


## Notes: ##
Using geojson from https://gis-txdot.opendata.arcgis.com/datasets/texas-state-boundary/explore
